packages/manager/modules/anthos/src/dashboard/general-information/general-information.routing.js

```diff
@@ -0,0 +1,26 @@
+export default /* @ngInject */ ($stateProvider) => {
+  const stateName = 'anthos.dashboard.general-information';
+  $stateProvider.state(stateName, {
+    url: '/general-information',
+    views: {
+      anthosTenantView: 'anthosDashboardGeneralInformation',
+    },
+    resolve: {
+      breadcrumb: () => false,
+
+      goBack: /* @ngInject */ (tenant, goToTenant) => (message, type) =>
+        goToTenant(tenant, message, type),
+
+      goToRenameService: /* @ngInject */ ($state, serviceName) => () =>
+        $state.go('anthos.dashboard.general-information.rename-service', {
+          serviceName,
+        }),
+
+      openAnthosConsole: /* @ngInject */ ($window, tenant) => () =>
+        $window.open(tenant.accessUrl, '_blank'),
```

Comment:
Is it possible to use `<a>` tag with `"oui-button"` class in HTML to avoid using `$window` here ?

Reply:
Done